kubeflow/trainer/backends/container/utils.py

```diff
@@ -150,38 +150,6 @@ def aggregate_status_from_containers(container_statuses: list[str]) -> str:
     return UNKNOWN
 
 
-def resolve_image(runtime: types.Runtime) -> str:
```

Comment:
@Fiona-Waters It looks like we don't need image resolver, since we fallback to the default runtime in case we can't get it online: https://github.com/kubeflow/sdk/blob/a9c5244abe6299c8b26461b10b70546283beba79/kubeflow/trainer/backends/container/runtime_loader.py#L314

Is that correct ?